benchmarks/benchmarks/bench_nexus.py

```diff
@@ -879,28 +808,47 @@ def start(self) -> None:
         validator.validate(self.filtered_hubs())
         self._release()
 
+    def _get_index_data(self, env_to_files: Mapping[EnvDescription, Iterable[Path]]) -> Dict[str, Any]:
+        return {
+            "env_files": {
+                env_description.key: [
+                    file.name for file in files
+                ]
+                for env_description, files in env_to_files.items()
+            },
+        }
+
     def _release(self):
         hub_to_director_to_env = self.load_directors(self.filtered_hubs())
         for hub_description, env_to_director in hub_to_director_to_env.items():
             env_with_accessor = [
                 (env_description, director.make_accessor())
                 for env_description, director in env_to_director.items()
             ]
+            env_to_accessor = {
+                env_description: accessor for env_description, accessor in env_with_accessor
+            }
             env_to_files = {
                 env_description: [
                     accessor.bench_result_file(accessor.get_id(schema))
                     for schema in accessor.schemas
                 ]
                 for env_description, accessor in env_with_accessor
             }
+
             with ZipFile(
                 file=RELEASE_DATA / f"{hub_description.key}.zip",
                 mode="w",
                 compression=ZIP_BZIP2,
                 compresslevel=9,
             ) as release_zip:
-                for file_path in chain.from_iterable(env_to_files.values()):
-                    release_zip.write(file_path, arcname=file_path.name)
+                for env in env_to_files:
+                    accessor = env_to_accessor[env]
+                    bench_operator = operator_factory(accessor, sqlite=bool(self.sqlite))
+                    bench_results = bench_operator.get_all_bench_results()
+
+                    for file_path, data in zip(env_to_files[env], bench_results):
+                        release_zip.writestr(file_path.name, data)
```

Comment:
This code looks very strange at the moment. At the beginning env_to_files is created, but then files is unused.
When you use sqlite to store data, there are no bench result files. As I said above, the bench_result_file method should go away from BenchAccessor.